package.json

```diff
@@ -19,7 +19,8 @@
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "swr": "^1.2.2",
-    "ts-jest": "^28.0.2",
+    "ts-jest": "^28.0.3",
```

Comment:
should all the testing packages be under devDependencies?

Reply:
yes. good catch